src/merlin.ml

```diff
@@ -48,18 +48,16 @@ let dot_merlin sctx ~dir ~scope ({ requires; flags; _ } as t) =
       requires &&& flags
       >>^ (fun (libs, flags) ->
         let ppx_flags = ppx_flags sctx ~dir ~scope ~src_dir:remaindir t in
-        let internals, externals =
-          List.fold_left libs ~init:([], []) ~f:(fun (internals, externals) ->
+        let libs =
+          List.fold_left libs ~init:[] ~f:(fun acc ->
+            let nice_path = Path.reach ~from:remaindir in
```

Comment:
I'm not sure `nice` gives the right idea. The idea is more to serialize an internal representation of a path. Maybe `serialize_path`?